exe/proteus

```diff
@@ -35,6 +38,13 @@ def configure(opts)
       }
     )
   end
+  # sanity check variables before proceeding to login!
+  if @config.url == "nil" ||
+     @config.username == "nil" ||
+     @config.password == "nil" ||
+     @config.viewid == "nil"
```

Comment:
Do these checks work? I wouldn't expect to have `nil` in quotes. Maybe `@config.url.nil?`, etc

Reply:
I removed quotes around nil and the checks work.  I also considered appending the individual errors into an array and looping through the array.